src/applications/simple-forms/21-0845/pages/organizationReps.js

```diff
@@ -1,60 +1,48 @@
 import React from 'react';
 
-import ArrayField from 'platform/forms-system/src/js/fields/ArrayField';
-import OrgRepsViewField from '../components/OrgRepsViewField';
-
 export default {
   uiSchema: {
     'ui:title': (
       <>
         <p className="vads-u-font-size--lg">Organization’s representatives</p>
         <p className="vads-u-margin-top--4 vads-u-margin-bottom--2 vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal vads-u-color--gray-dark">
-          List one or more people from the organization who we can share your
-          information with.
+          List at least one person from the organization who we can release your
+          information to.
         </p>
       </>
     ),
-    organizationRepresentatives: {
-      'ui:field': ArrayField,
-      'ui:options': {
-        itemName: 'representative',
-        viewField: OrgRepsViewField,
-        keepInPageOnReview: true,
-        useDlWrap: true,
-        showSave: false,
-        reviewMode: true,
-        customTitle: ' ',
-        confirmRemove: true,
-        confirmRemoveDescription:
-          'This will remove the Representative associated with the Organization.',
-      },
-      items: {
-        representativeName: {
-          'ui:title': 'Name of representative',
-          'ui:required': () => true,
-          'ui:errorMessages': {
-            required: 'Please provide the Representative’s name',
-          },
-        },
+    organizationRepresentative: {
+      'ui:title': (
+        <div className="vads-u-font-weight--normal vads-u-font-size--base vads-u-margin-bottom--2">
+          <span>
+            Name of representative{' '}
+            <span className="custom-required-span hide-on-review-page">
+              (*Required)
```

Comment:
I know this has been done in some past PRs, but what is the reason for manually rendering a Required span? And why does it get hidden on the review page? Does it have to do with that header level issue? Is that `hide-on-review-page` class from platform? Or did you create it?

Reply:
@ndsprinkle , yeah, it's kinda ugly...

Platform's reported a11y-bug was that the `'ui:description'`s were getting bypassed by screen-readers.  I had to move those description-contents into the `'ui:title'`s in order for them to get read.  This also meant I had to add my own required-span _between_ the custom-title and custom-description, and hide it on Review-page.

I'm gonna refactor the custom `'ui:title'`s, and might be able to avoid my custom 'hide-on-review-page' hack.

I'll update here when refactors are pushed.

Reply:
Refactors done.  No more `hide-on-review-page` classhooks; now using `'ui:reviewField'` instead to separate custom-labels from review-page labels.